opm/simulators/wells/SingleWellState.cpp

```diff
@@ -156,8 +156,14 @@ double SingleWellState::sum_solvent_rates() const {
 void SingleWellState::update_producer_targets(const Well& ecl_well, const SummaryState& st) {
     const double bhp_safety_factor = 0.99;
     const auto& prod_controls = ecl_well.productionControls(st);
-    if (prod_controls.hasControl(Well::ProducerCMode::THP))
+    if (prod_controls.hasControl(Well::ProducerCMode::THP)) {
         this->thp = prod_controls.thp_limit;
+        // For THP controlled wells we initialize with some non-zero rates
+        // to get a more resonable starting point for the lookup in the vpf table.
+        for (int p = 0; p < pu.num_phases; ++p) {
+            this->surface_rates[p] = -1.0;
```

Comment:
So this is effectively 1 cubic metre per second, right?  If so, maybe you can add a comment to that effect?

Reply:
The rate should be in the range of the vfp tables FLO entries. Since they are typically in SM3/day I think I would like to change it to 100 cubic meter pr day. It shouldn't be too sensitive to the actual number as long as it is not too close to zero. And yes it make sense to a comment.

Reply:
more testing indicates that the convergence (or more the lack of convergence) is more sensitive to the choice of this more or less random rate that I anticipated. i.e. i need to go back and investigate more so I converted it to draft.

Reply:
I think it makes sense because the ratios also matters a lot when coming to VFP table search.